client/src/components/app/App.js

```diff
@@ -16,8 +16,8 @@ class App extends Component {
         this.state = {
             apiVersion: null,
             version,
-            minApiVersion: '0.0.0',
-            maxApiVersion: '1.0.0',
+            minApiVersion: '3.0.0',
+            maxApiVersion: '4.0.0',
```

Comment:
Is a max version necessary? What will happen when we get above version 4.0.0?

Reply:
yeah i almost just removed the whole thing tbh...

Reply:
yeah it is necessary or yeah it is _not_ necessary?

Reply:
I think it might be a bit overkill the whole thing.

Reply:
The hope was that you would find out that you need to update your extension when it goes out of date. Perhaps as you say, just drop max and keep min

Reply:
If the goal is to detect if something is out of date; then it might be more sane to do a proper semver check: https://www.npmjs.com/package/semver